examples/timestamping/backend/src/api.rs

```diff
@@ -82,6 +93,7 @@ impl PublicApi {
         })
     }
 
+    /// Wires these endpoints to public API scope of given `ServiceApiBuilder`.
```

Comment:
Not really clear what 'these endpoints' are. If this description comes after the description of the endpoints, perhaps  'Wires the above endpoints' would be better.
Also, 'of the given' would sound better.